src/main/java/org/itc/AppConfig.java

```diff
@@ -5,9 +5,16 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.core.env.Environment;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.jdbc.repository.config.EnableJdbcRepositories;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+
+import javax.sql.DataSource;
 
 @Configuration
 @ComponentScan(basePackages = "org.itc")
+@EnableJdbcRepositories
 public class AppConfig {
```

Comment:
Si van a trabajar con spring framework, deben extender de AbstractJdbcConfiguration (Hacer el respectivo import)

```suggestion
public class AppConfig extends AbstractJdbcConfiguration {
```